lib/ssh_scan/database/mongo.rb

```diff
@@ -0,0 +1,40 @@
+require 'mongo'
+
+Mongo::Logger.logger.level = ::Logger::FATAL
+
+module   SSHScan
+  module Database
+    class MongoDb
+      def initialize(opts = {})
+        name = opts["database"]["name"] || "ssh_scan"
```

Comment:
Same here, you will end up with NoMethod errors if the database key is not present.